components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -575,9 +575,10 @@ void RewardsServiceImpl::OnPublisherStateSaved(
 void RewardsServiceImpl::SavePublisherInfo(
     std::unique_ptr<ledger::PublisherInfo> publisher_info,
     ledger::PublisherInfoCallback callback) {
+  ledger::PublisherInfo info_copy = *publisher_info;
```

Comment:
not sure why this fixes it because it shouldn't actually change anything, but I can't repro after this change. Maybe a compiler bug?